tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -321,6 +319,12 @@ def __init__(
             pytorch_backend_config.torch_compile_piecewise_cuda_graph
             and not self.enable_attention_dp)
 
+        self._piecewise_cuda_graph_num_tokens = pytorch_backend_config.torch_compile_piecewise_cuda_graph_num_tokens if pytorch_backend_config.torch_compile_piecewise_cuda_graph_num_tokens else pytorch_backend_config.cuda_graph_batch_sizes if pytorch_backend_config.cuda_graph_batch_sizes else []
```

Ruff (0.12.2)

Line too long (296 > 120)

(E501)